src/app/components/MediaLoader/index.tsx

```diff
@@ -249,6 +249,9 @@ const MediaLoader = ({ blocks, className, embedded }: Props) => {
             orientation === 'portrait' && styles.portraitFigure(embedded),
             orientation === 'landscape' && styles.landscapeFigure,
           ],
+          playerConfig?.ui?.skin === 'audio' &&
+            hasPlaceholder &&
```

Comment:
Not sure if this combination can/should happen. The audio UI won't have a placeholder, so we'll need to prevent it from showing with something like:
```
  const hasPlaceholder = Boolean(
    showPlaceholder && placeholderSrc && playerConfig?.ui?.skin !== 'audio',
  );
```

Or by setting the `placeholderSrc` to `null/undefined` in the config for `clipMedia` if its an audio clip.

Reply:
The second option is probably better as it isolates the logic to the particular config type.

Reply:
ahh sorry i thought that the audio did have a placeholder based off the screenshots in the ticket , is this incorrect? the data also came through with a placeholder image - but is this something we dont want grabbing ? would it just always be that flat bbc logo - https://ichef.test.bbci.co.uk/images/ic/512xn/p01vkjg8.png.webp
<img width="290" alt="Screenshot 2024-10-04 at 15 38 25" src="https://github.com/user-attachments/assets/1a6e99ed-2f86-4c1b-ab49-c10bca7ff407">